tests/test_driver.py

```diff
@@ -223,6 +291,47 @@ def test_retrieve(self, driver, block_with_alice_transaction):
         assert block
 
 
+class TestBlocksEndppointMultiple:
+
+    @mark.asyncio
+    @mark.parametrize("timeout", [0.5])
+    async def test_get_timeout(self, timeout, event_loop,
```

Comment:
the name of this (and the tests below) are a bit confusing because you don't get/receive a timeout/loop